### DISCUSSION TITLE
文件上传后是否会一直存在本地？

### DISCUSSION THREAD

**Discussion by flyingwasabi** [AlistGo/alist#1368] [Q&A]
请问文件在上传到云端（如google drive）之后是否会在储存容量不足时自动删除本地拷贝？

如果能自动删除本地拷贝只留云端的那一份，是否可以用来作为google drive的中转？想安装在raspberry pi上所以并没有很大的本地容量可以使用。

谢谢

**Comment by Jue-Xu** [comment]
同问。应该是不会一直在本地。可以实践（测试）一下
